index.html

```diff
@@ -17,11 +17,20 @@
         <div class="nav-logo">
           <a href="index.html"><p>Welcome</p></a>
         </div>
-        <div class="hamburger">
+        <div class="hamburger" id="hamburger">
           <img src="./img/hamburger.png" alt="hamburger menu" />
         </div>
+        <div class="close-x" id="close-x">
+          <img src="./img//close-x.png" alt="hamburger menu" />
+        </div>
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.

Reply:
Thank you.
I have implemented all the optional reviews into my code.